framework/packages/abstract-client/src/account.rs

```diff
@@ -591,6 +687,22 @@ impl<Chain: CwEnv> Account<Chain> {
         // We install only one module
         Addr::unchecked(module_address)
     }
+
+    pub(crate) fn module<T: RegisteredModule + From<Contract<Chain>>>(
```

Comment:
So we only want `Application.module` and not `Account.module`?

Reply:
> So we only want `Application.module` and not `Account.module`?

I thought that when we're using `.module` we are expecting to get dependency of an `Application`. I'm okay to expose it on `Account` as well, if that would make sense. User still can get module from an `Application` with `.module` or `Deref`